pkg/tfgen/generate_schema.go

```diff
@@ -58,151 +55,6 @@ type schemaGenerator struct {
 	language Language
 }
 
-type schemaNestedType struct {
```

Comment:
There's some code motion here over some tricky code, could we please separate the commits that move the code from the commits that meaningfully edit it for easier review, to be able to see the new code. Thanks a lot!